src/components/Card.tsx

```diff
@@ -0,0 +1,54 @@
+import '../styles/Card.scss';
+import { Link } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
+export interface CardProps {
+    imageSrc: string;
+    href: string;
+    title: string;
+    body: string;
+    showInitialText: boolean;
+    textHideable: boolean;
+    imageOnRight: boolean;
+    isTimelineCard: boolean;
+}
+
+export function Card({ imageSrc, href, title, body, showInitialText, textHideable, imageOnRight, isTimelineCard }: CardProps) {
+
+    const [textVisible, setTextVisible] = useState(showInitialText);
+
+    function changeToggleText(){
```

Comment:
this is no longer changing toggle text, it may be worth calling it something like `toggleTextVisible` :)

Reply:
updated name